projects/ngx-formentry/src/components/ngx-remote-select/ngx-remote-select.component.ts

```diff
@@ -65,6 +66,13 @@ export class RemoteSelectComponent implements OnInit, ControlValueAccessor {
 
   ngOnInit() {
     this.loadOptions();
+    // check if there is enough space at the bottom of the ng-select component
+    const selectElement = document.querySelector('ng-select');
+    const selectBoundingRect = selectElement.getBoundingClientRect();
+    const bottomSpace = window.innerHeight - selectBoundingRect.bottom;
```

Comment:
Outside of the context of the test app, I'm not sure `window.innerHeight` is the right property, since it assumes the form is the full height of the window. It seems like it would be better to compare the `bottom` of the bounding rectangle for the select element with the bottom for the bounding rectangle of the `my-app-root` element (since that's the root of the form tree). That way, if the form engine is rendered in a confined space, it will still continue to work.